#goal definition
int32 id
float32 pos_x
float32 pos_y
float32 pos_z
---
#result definition
int32 result
---
#feedback
int32 feedback
